### PR TITLE
Steering Committee Membership: chrisohaver

### PR DESCRIPTION
**Pull request by chrisohaver**
Terms are ending next week (#6327). I am proposing that I remain on the steering committee through the next term (ending 9/2024).

Please vote per GOVERNANCE.md rules.

### CODE REVIEW

**Review by SuperQ — APPROVED** [review]

**Review by jameshartig — APPROVED** [review]

File: CODEOWNERS [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # 5 steering committee members
-# @chrisohaver, term ends 26 Sept 2023
+# @chrisohaver, term ends 26 Sept 2024
```
Comment (SuperQ) [inline CODEOWNERS]: ISO 8601 :grin:

```suggestion
# @chrisohaver, term ends 2024-09-26
```

**Review by SuperQ — APPROVED** [review]
:+1:

**Review by johnbelamaric — APPROVED** [review]

**Comment by johnbelamaric** [comment]
:+1:

**Review by bradbeam — APPROVED** [review]

**Review by greenpau — APPROVED** [review]

**Review by greenpau — APPROVED** [review]

**Review by Isolus — APPROVED** [review]

**Comment by stp-ip** [comment]
+1

**Review by Tantalor93 — APPROVED** [review]

**Review by dilyevsky — APPROVED** [review]
👍

**Review by zouyee — APPROVED** [review]

**Pull request merged**